### DISCUSSION TITLE
volume or ivolume in GDL

### DISCUSSION THREAD

**Discussion by brandy125** [gnudatalanguage/gdl#1648] [Q&A]
Is there any program in GDL to display a 3-D dataset like ivolume does?
Example in IDL:

```
IDL> a=dist(500)
IDL> b=rebin(a,500,500,50)
IDL> ivolume,b
```
Then Operations->Volume->Isosurface->ok

**Comment by GillesDuvert** [comment]
Afraid no :frowning_face:
